agentops/llm_tracker.py

```diff
@@ -227,6 +227,34 @@ async def patched_function(*args, **kwargs):
         # Override the original method with the patched one
         completions.AsyncCompletions.create = patched_function
 
+    def override_litellm_completion(self):
+        import litellm
+
+        original_create = litellm.completion
```

Comment:
Wasn't the issue with litellm that it wasn't sending model provider? Hence, causing us to not be able to look them up with tokencost

Reply:
The point of this PR was to capture that model provider. It was stripped out in the OpenAI call but not the litellm.completion call hence why we're instrumenting it